terms/ngs/directionalBSseqLibrary.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-ngs.directionalBSseqLibrary-0.0.1",
+  "description": "Indicates whether or not the library is directional",
```

Comment:
Does the description need to clarify that this is only for bisulfite seq, or do we think that's clear enough from the key name?

Reply:
@karawoo I've clarified the description to indicate that the library is for bisulfiteSeq.